src/app.js

```diff
@@ -1,8 +1,10 @@
 import React from 'react';
-import ReactDOM from 'react-dom'
+import ReactDOM from 'react-dom';
+import {StyleRoot} from "radium";
 import MainPage from './components/MainPage';
-import 'normalize.css/normalize.css'
```

Comment:
I removed normalize and my overall styles SCSS file as they are no longer being used